templates/miq-template.yaml

```diff
@@ -691,10 +771,14 @@ parameters:
   displayName: Memcached Slab Page Size
   description: Memcached size of each slab page.
   value: 1m
+- name: POSTGRESQL_CONFIG_DIR
+  displayName: PostgreSQL Configuration Overrides
+  description: Directory used to store PostgreSQL configuration overrides.
+  value: "/var/lib/pgsql/conf.d"
 - name: POSTGRESQL_MAX_CONNECTIONS
   displayName: PostgreSQL Max Connections
   description: PostgreSQL maximum number of database connections allowed.
-  value: '100'
+  value: '1000'
 - name: POSTGRESQL_SHARED_BUFFERS
```

Comment:
@carbonin I noticed our default MIQ overrides had it at 1000 and went ahead for the change. Let me know if you prefer a different PR for this setting and I'll revert it.